components/KiboPrice.vue

```diff
@@ -0,0 +1,92 @@
+<template functional>
+  <div>
+    <div
+      :class="[data.class, data.staticClass, 'kibo-price']"
+      :style="[data.style, data.staticStyle]"
+      v-bind="data.attrs"
+      v-on="listeners"
+    >
+      <!--@slot Custom regular price -->
+      <slot name="regular" v-bind="{ props }">
+        <span
+          :class="
+            !props.regular || props.special
+              ? 'display-none'
+              : props.small
+              ? `kibo-price__regular--small`
+              : `kibo-price__regular`
+          "
+        >
+          {{ props.regular }}
+        </span>
+      </slot>
+      <!--@slot Custom old price (value from regular)-->
+      <slot name="old" v-bind="{ props }">
+        <del
+          :class="
+            !props.special
+              ? 'display-none'
+              : props.small
+              ? `kibo-price__old--small`
+              : `kibo-price__old`
+          "
+        >
+          {{ props.regular }}
+        </del>
+      </slot>
+      <!--@slot Custom special price -->
+      <slot name="special" v-bind="{ props }">
+        <ins
+          :class="
+            !props.special
+              ? 'display-none'
+              : props.small
+              ? `kibo-price__special--small`
+              : `kibo-price__special`
+          "
+        >
+          {{ props.special }}
+        </ins>
+      </slot>
+    </div>
+    <slot name="promo-success" v-bind="{ props }">
+      <ins v-if="props.special" class="kibo-promo-success">
```

Comment:
Why are the coupon codes added into the Price component? 
These aren't global and only shown on the cart

Reply:
extracted to parent level.